packages/remix-node/fetch.ts

```diff
@@ -72,5 +78,11 @@ export const fetch: typeof webFetch = (
     ...init,
   };
 
-  return webFetch(info, init as RequestInit);
+  return webFetch(info, init as RequestInit)
+    .then((res) => {
+      return getReturnableResponse(res);
+    })
+    .catch((res) => {
+      return getReturnableResponse(res);
+    });
```

Comment:
This could have a smaller footprint if we do

```suggestion
    .then(getReturnableResponse)
    .catch(getReturnableResponse);
```